### DISCUSSION TITLE
Dark mode support with a quick switch

### DISCUSSION THREAD

**Discussion by brnquester** [lucky-media/landtamic#33] [Ideas]
Are there any plans to support dark mode for the user? Similar to what you guys offer on your agency website. https://www.luckymedia.dev

**Comment by lokmanm** [comment]
Hey @brnquester,

Sorry for getting back to you so late. Adding Dark Mode to Landtamic means we'd have to pretty much overhaul the color palette and almost redesign everything.

I can't spill the beans on our future plans, but we might think about it.

Thanks!

**Reply by brnquester** [reply]
Hi @lokmanm.

No problem. Thanks for getting back to me. Great job you did with Landtamic, it has a lot of potential. 

Keep up the great work.
